addon/adapters/user.js

```diff
@@ -1,4 +1,20 @@
 import OsfAdapter from './osf-adapter';
+import Ember from 'ember';
 
 export default OsfAdapter.extend({
+    findHasMany(store, snapshot, url, relationship) {
```

Comment:
Unless I have to, I'm going to try to avoid loading the entire context for this in order to completely answer the question, but if I need to, we can have a discussion. That being said, I am okay with either: 1) allowing `?embed=root,parent`; or 2) allowing headers that specify what is embedded (as long as conflicts either `Conflict` or do the superset, in this case). If either of those solutions make this easier, go for it. If more understanding is required, let's set aside some time.